query-engine/connector-test-kit/src/test/scala/util/ConnectorConfig.scala

```diff
@@ -123,6 +124,22 @@ object ConnectorConfig {
     }
   }
 
+  lazy val $cockroachdb_Host = {
```

Comment:
The variable name should be without `$`. Only when you reference it in an interpolated string like above you use the `$`.